src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -189,6 +190,13 @@ const routes: Routes = [
         path: 'multi-cluster',
         component: MultiClusterComponent
       },
+      {
+        path: 'manage-clusters',
+        component: MultiClusterListComponent,
+        data: {
+          breadcrumbs: 'Multi-Cluster/Manage Clusters'
+        }
+      },
```

Comment:
this should be a child component of the `multi-cluster` path, not a separate entity